app/src/main/java/com/pajato/android/gamechat/fragment/TTTFragment.java

```diff
@@ -234,4 +228,51 @@ private boolean checkNotFinished() {
         return true;
     }
 
+    /**
```

Comment:
It may be time to start thinking about abstracting the TTT player.  For example, each player will be one of "X" or "O", may have an association with a User, will have an icon, active or inactive indicator, etc.  Abstracting this into a Player class may make the code simpler.